apps/src/templates/RegionalPartnerSearch.jsx

```diff
@@ -303,9 +304,32 @@ class RegionalPartnerSearch extends Component {
                   collection => collection.workshops.length === 0
                 ) && <div>Workshop details coming soon!</div>}
```

Comment:
maybe a weird quirk, if there are no workshops at all, it'll show this text and also the CSA text

Reply:
Strong point––I'll check with Tess what we want to do in that situation

Reply:
She says it's not a huge deal since this won't happen next year, but I'll add something for it

Reply:
Just tweaked it––new screenshots for workshops, and then if there are no workshops
<img width="964" alt="Screen Shot 2022-03-04 at 2 21 53 PM" src="https://user-images.githubusercontent.com/9142121/156828630-4a8783dd-c25b-4152-a972-10a6db5b12e6.png">
<img width="809" alt="Screen Shot 2022-03-04 at 2 22 46 PM" src="https://user-images.githubusercontent.com/9142121/156828644-07a7c28f-cb11-44ad-b435-7c9f33204d11.png">